### DISCUSSION TITLE
不错的工作，但好像识别率有些问题

### DISCUSSION THREAD

**Discussion by zhaoyun0071** [OleehyO/TexTeller#37] [feedback]
我尝试了扫描版pdf里的公式识别，正确率不太行

**Comment by OleehyO** [comment]
TexTeller是用渲染出来的图片训练的，扫描版的pdf可能确实不行。

**Comment by zhaoyun0071** [comment]
> TexTeller是用渲染出来的图片训练的，扫描版的pdf可能确实不行。

https://github.com/NormXU/nougat-latex-ocr
这个就可以，可能nougat底模强大吧
